ceph.spec.in

```diff
@@ -198,10 +198,10 @@ BuildRequires:	python%{_python_buildid}-requests
 BuildRequires:	python%{_python_buildid}-six
 BuildRequires:	python%{_python_buildid}-tox
 BuildRequires:	python%{_python_buildid}-virtualenv
-%if 0%{?fedora} || 0%{?rhel}
+%if 0%{?rhel} == 7
 BuildRequires:  pyOpenSSL%{_python_buildid}
 %endif
-%if 0%{?suse_version}
+%if 0%{?fedora} || 0%{?suse_version} || 0%{?rhel} == 8
```

Comment:
how about use `% else` instead ? to be consistent with 

https://github.com/ceph/ceph/blob/d7a279f7599e6917977f4770432fe3da7cfc47ed/ceph.spec.in#L473-L477